### PR TITLE
Add label to semantic models

### LINKED ISSUE #4114: A label can be now specified on SemanticModel nodes, Dimensions, Entities, and Measures dbt-core PR #8646
A label can be now specified on SemanticModel nodes, Dimensions, Entities, and Measures

https://docs.getdbt.com/docs/build/semantic-models

This is similar to how it's already [documented for metrics](https://docs.getdbt.com/docs/build/metrics-overview#:~:text=Optional-,label,-The%20display%20name)

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Addresses issue https://github.com/dbt-labs/docs.getdbt.com/issues/4114

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/build/semantic-models.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Semantic models have 6 components and this page explains the definitions with so
 | [Primary Entity](#primary-entity) | If a primary entity exists, this component is Optional. If the semantic model has no primary entity, then this property is required. | Optional |
 | [Dimensions](#dimensions) | Different ways to group or slice data for a metric, they can be `time` or `categorical` | Required |
 | [Measures](#measures) | Aggregations applied to columns in your data model. They can be the final metric or used as building blocks for more complex metrics | Optional |
+| Label | The display name for your `node`, `dimension`, `entity`, and/or `measures` | Optional |
```
Comment (runleonarun) [inline website/docs/docs/build/semantic-models.md]: Should we specify SemanticModel node? Is this label visible in downstream tools like [labels in metrics](https://docs.getdbt.com/docs/build/metrics-overview#:~:text=Optional-,label,-The%20display%20name)?

**Review by runleonarun — APPROVED** [review]
Approving with some questions that may or may not need to be addressed.

Was this backported or does it need to be versioned?

File: website/docs/docs/build/semantic-models.md (hunk 1 above)
Comment (matthewshaver) [inline website/docs/docs/build/semantic-models.md]:
```suggestion
| Label | The display name for your semantic model `node`, `dimension`, `entity`, and/or `measures` | Optional |
```

**Comment by runleonarun** [comment]
🚢 it!

**Pull request merged**